<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulación Estación de Servicio</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="surtidoresLabel">
        <property name="text">
         <string>Número de Surtidores</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="empleadosLabel">
        <property name="text">
         <string>Número de Empleados</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="nSurtidores"/>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="nEmpleados"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="comandoLabel">
        <property name="text">
         <string>Comando</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="comandoEditText"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="iniciarButton">
        <property name="text">
         <string>Iniciar Simulación</string>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="salidaEditText">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="limpiarButton">
      <property name="text">
       <string>Limpiar</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>iniciarButton</tabstop>
  <tabstop>salidaEditText</tabstop>
  <tabstop>limpiarButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>limpiarButton</sender>
   <signal>clicked()</signal>
   <receiver>salidaEditText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
